src/lib/txn.js

```diff
@@ -9,6 +9,9 @@ import { ledgerSignTransaction } from './ledger';
 import { trezorSignTransaction } from './trezor';
 import { WALLET_TYPES, addHexPrefix } from './wallet';
 
+const kAverageBlockTimeMs =
+  process.env.NODE_ENV === 'development' ? 1000 : 13000;
```

Comment:
I'd probably just check every five seconds in a live environment, to reduce response times?

Reply:
sgtm - was using the old 13000 magic number, but I like the idea of 5000